packages/slate-dev-environment/src/index.js

```diff
@@ -91,6 +91,43 @@ if (isBrowser) {
   }
 }
 
+/**
+ * Array of regular expression matchers and their API version
+ *
+ * @type {Array}
+ */
+
+const ANDROID_API_VERSIONS = [
+  [/^9([.]0|)/, 28],
+  [/^8[.]1/, 27],
+  [/^8([.]0|)/, 26],
+  [/^7[.]1/, 25],
+  [/^7([.]0|)/, 24],
+  [/^6([.]0|)/, 23],
+  [/^5[.]1/, 22],
+  [/^5([.]0|)/, 21],
+  [/^4[.]4/, 20],
+]
+
+/**
+ * get the Android API version from the userAgent
+ *
+ * @return {number} version
+ */
+
+function getAndroidApiVersion() {
+  if (os !== 'android') return null
+  const { userAgent } = window.navigator
```

Comment:
I just realized this runs on module load, and I'm wondering if this code will ever be run during SSR. In that case the use of `window` has to be guarded.

Reply:
@wmertens Nice catch! Didn't even think about that.

Reply:
@wmertens Nice catch; however, `window.navigator` will never get called because `os !== 'android'` will be true if we aren't in a browser so I think we're still okay.

Reply:
Ah yes of course - unless we start running render farms on cheap android tv sticks ;)